src/helpers/decodeInterlaceAdam7.ts

```diff
@@ -24,7 +24,7 @@ export function decodeInterlaceAdam7(params: DecodeInterlaceNullParams) {
     { x: 0, y: 1, xStep: 1, yStep: 2 }, // Pass 7
   ];
 
-  const bytesPerPixel = (channels * depth) / 8;
+  const bytesPerPixel = Math.ceil(channels * depth) / 8;
```

Comment:
I'm surprised by the location of the closing parenthesis. Aren't `channels` and `depth` both integers? In that case `Math.ceil` would not change the result unless you call it after the division by 8.